docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md

```diff
@@ -69,16 +88,85 @@ To add items to a shopping list, send the request:
+
+```json
+{
+    "data": {
+        "type": "shopping-list-items",
+        "attributes": {
+            "sku": "093_24495843",
+            "quantity": 3,
+            "productConfigurationInstance": {
+                "displayData": "{\"Preferred time of the day\": \"Afternoon\", \"Date\": \"9.09.2050\"}",
+                "configuration": "{\"time_of_day\": \"4\"}",
+                "configuratorKey": "installation_appointment_test",
+                "isComplete": true,
+                "quantity": 3,
+                "availableQuantity": 4,
+                 "prices": [
+                	  {
+                        "priceTypeName": "DEFAULT",
+                        "netAmount": 23434,
+                        "grossAmount": 42502,
+                        "currency": {
+                            "code": "EUR",
+                            "name": "Euro",
+                            "symbol": "€"
+                        },
+                        "volumePrices": [
+                            {
+                                "netAmount": 150,
+                                "grossAmount": 165,
+                                "quantity": 5
+                            },
+                            {
+                                "netAmount": 145,
+                                "grossAmount": 158,
+                                "quantity": 10
+                            },
+                            {
+                                "netAmount": 140,
+                                "grossAmount": 152,
+                                "quantity": 20
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    }
+}
+```
+
+</details>
 
 | ATTRIBUTE | TYPE | REQUIRED | DESCRIPTION |
 | --- | --- | --- | --- |
-| quantity | Integer | ✓ | Quantity of the product to add. |
-| sku | String | ✓ | SKU of the product to add. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) are allowed. |
+| quantity | Integer | &check; | Quantity of the product to add. |
+| sku | String | &check; | SKU of the product to add. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) and [configurable products](/docs/scos/user/features/{{site.version}}/configurable-product-feature-overview.html) are allowed. |
+| productOfferReference | String | &check; | Unique identifier of the product offer. |
+|merchantReference| String | &check; | Unique identifier of the merchant. |
+| productConfigurationInstance.displayData  | Array  |&check; | Array of variables that are proposed for a Storefront user to set up in the configurator.  |
+| productConfigurationInstance.configuration  | Array  | &check; | Default configurable product configuration.  |
```

Comment:
```suggestion
| productConfigurationInstance.configuration  | Array  | &check; | Default configuration of the configurable product.  |
```
As Default configurable product configuration might be misunderstood as configuration of a default configurable product"